Build fails due to npm issue

**Description:**
Ballerina Integrator build fails with the following exception

```
ERROR  Command failed: npm list --production --parseable --depth=99999
npm ERR! extraneous: generator-code@1.2.6 /build/jenkins-home/jobs/ballerina-integrator-release/workspace/vscode/extension/node_modules/generator-code
npm ERR! extraneous: vsce@1.66.0 /build/jenkins-home/jobs/ballerina-integrator-release/workspace/vscode/extension/node_modules/vsce
npm ERR! extraneous: yo@3.1.0 /build/jenkins-home/jobs/ballerina-integrator-release/workspace/vscode/extension/node_modules/yo


npm ERR! Linux 4.15.0-46-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "package"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! ballerina-integrator@0.9.0 package: `vsce package`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ballerina-integrator@0.9.0 package script 'vsce package'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the ballerina-integrator package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     vsce package
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs ballerina-integrator
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls ballerina-integrator
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /build/jenkins-home/jobs/ballerina-integrator-release/workspace/vscode/extension/npm-debug.log
[ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:404)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:166)
	at org.codehaus.mojo.exec.ExecMojo.executeCommandLine(ExecMojo.java:804)
	at org.codehaus.mojo.exec.ExecMojo.executeCommandLine(ExecMojo.java:751)
	at org.codehaus.mojo.exec.ExecMojo.execute(ExecMojo.java:313)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:101)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:209)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:84)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:59)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.singleThreadedBuild(LifecycleStarter.java:183)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:161)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:320)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:156)
	at org.jvnet.hudson.maven3.launcher.Maven3Launcher.main(Maven3Launcher.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchStandard(Launcher.java:329)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:239)
	at org.jvnet.hudson.maven3.agent.Maven3Main.launch(Maven3Main.java:181)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at hudson.maven.Maven3Builder.call(Maven3Builder.java:139)
	at hudson.maven.Maven3Builder.call(Maven3Builder.java:70)
	at hudson.remoting.UserRequest.perform(UserRequest.java:212)
	at hudson.remoting.UserRequest.perform(UserRequest.java:54)
	at hudson.remoting.Request$2.run(Request.java:369)
	at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:72)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

```

--- a/pom.xml
+++ b/pom.xml
@@ -84,7 +84,6 @@
             </executions>
             <configuration>
                <executable>${project-home}/resources/run${script.extension}</executable>
-               <skip>true</skip>
             </configuration>
          </plugin>
       </plugins>

--- a/resources/run.sh
+++ b/resources/run.sh
@@ -19,11 +19,11 @@
 # Build Ballerina Integrator Tests
 # ----------------------------------------------------------------------------
 
-HOME=`pwd`
-source $HOME/resources/config.properties
-if [ -e $HOME/completeTestResults.log ]
+BI_HOME=`pwd`
+source $BI_HOME/resources/config.properties
+if [ -e $BI_HOME/completeTestResults.log ]
 then
-    rm -rf $HOME/completeTestResults.log
+    rm -rf $BI_HOME/completeTestResults.log
 fi
 
 executionNameList=("healthcare-service" "exposing-several-services-as-a-single-service" 
@@ -51,7 +51,7 @@ echo ' _____         _
 
 #build healthcare service
 echo "Executing healthcare-service"
-cd $HOME/${executionPathList[0]}
+cd $BI_HOME/${executionPathList[0]}
 ballerina build healthcare > testResults
 if ((grep -q "[1-9][0-9]* failing" testResults) || ! (grep -q "Running tests" testResults))
 then
@@ -62,19 +62,19 @@ else
 fi
 echo -e "------End of executing ${executionNameList[0]} tests----- \n"
 ((count++))
-cat testResults >> $HOME/completeTestResults.log
+cat testResults >> $BI_HOME/completeTestResults.log
 
 # move cache to ballerina home
-mkdir -p /home/travis/.ballerina/bir_cache/wso2/healthcare/0.1.0
+mkdir -p $HOME/.ballerina/bir_cache/wso2/healthcare/0.1.0
 
-cp /home/travis/build/wso2/ballerina-integrator/docs/learn/backends/healthcare-service/target/caches/bir_cache/wso2/healthcare/0.1.0/healthcare.bir /home/travis/.ballerina/bir_cache/wso2/healthcare/0.1.0 
+cp $BI_HOME/docs/learn/backends/healthcare-service/target/caches/bir_cache/wso2/healthcare/0.1.0/healthcare.bir /$HOME/.ballerina/bir_cache/wso2/healthcare/0.1.0 
 
 # TODO: Count should be changed to 1 when issue (https://github.com/wso2/ballerina-integrator/issues/316) is fixed.
 count=0
 for i in "${executionPathList[@]}"; 
 do 
 	echo "Executing ${executionNameList[$count]}"
-	cd $HOME/${executionPathList[$count]}
+	cd $BI_HOME/${executionPathList[$count]}
 	ballerina build ${moduleList[$count]} > testResults
 	if ((grep -q "[1-9][0-9]* failing" testResults) || ! (grep -q "Running tests" testResults))
 	then
@@ -85,5 +85,5 @@ do
 	fi
 	echo -e "------End of executing ${executionNameList[$count]} tests----- \n"
 	((count++))
-	cat testResults >> $HOME/completeTestResults.log
+	cat testResults >> $BI_HOME/completeTestResults.log
 done;

